### DISCUSSION TITLE
Receiving no suggestions on MacOS and VS Code

### DISCUSSION THREAD

**Discussion by oliver-mentel** [community/community#32418] [Copilot Conversations]
Trying to use Copilot on a Mac with VS Code (extension version v1.44.6735). 
I tried  to re-install the plugin but still not receiving any code suggestions.

Any ideas why Copilot is not working?

**Comment by davecheney** [comment]
@oliver-mentel i'm sorry you are having difficulties. Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
